apollo-router/src/executable.rs

```diff
@@ -50,7 +52,12 @@ pub(crate) struct Opt {
     log_level: String,
 
     /// Reload configuration and schema files automatically.
-    #[clap(alias = "hr", long = "hot-reload", env = "APOLLO_ROUTER_HOT_RELOAD")]
+    #[clap(
+        alias = "hr",
+        long = "hot-reload",
+        env = "APOLLO_ROUTER_HOT_RELOAD",
```

Comment:
Are these new environment variables? If so, do we want/need to document them?

Reply:
It's not new ones, I just added a parameter to clap to correctly parse it under the hood